src/http.js

```diff
@@ -25,6 +25,7 @@ export default function http(url, request) {
   if (request.requestInterceptor) {
     request = request.requestInterceptor(request) || request
   }
+  request.credentials = 'same-origin';
```

Comment:
@buunguyen I would think that it would be done in general and not per request, unless that creates issues.

In cases where you're calling to a different origin (i.e. from petstore.swagger.io to a different host) are you _punished_ for setting this value, or is it just ignored?